site/plugins/gravatar/index.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+function gravatar(string $email, string|null $default = null): string {
+	$address = strtolower(trim($email));
+	$hash    = hash('sha256',$address);
+
+	return 'https://www.gravatar.com/avatar/' . $hash . '?d=' . $default;
+}
```

Comment:
`Suggestion:` Don't we maybe want to make this part of the discord plugin as we would need it in multiple repos and maybe also helpful for others?

Reply:
It felt a bit out of scope. I was wondering if it should be its own super small official plugin that can be used in multiple other places as well. But on the other hand I really also don't mind too much. Could be in the Author class as well.